extensions/kubernetes/spi/src/main/java/io/quarkus/kubernetes/spi/DekorateOutputBuildItem.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.kubernetes.spi;
+
+import java.util.List;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * Produce this build item to expose the Dekorate project and Dekorate session.
+ */
+public final class DekorateOutputBuildItem extends SimpleBuildItem {
```

Comment:
As there is nothing specific to dekorate in this build item. I would prefer a more generic name.

Reply:
It turns out that this is quite speicific to dekorate